ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/management-tech-support.j2

```diff
@@ -0,0 +1,24 @@
+{# Management tech-support #}
+{% if management_tech_support is arista.avd.defined %}
+!
+management tech-support
+{%     if management_tech_support.policy_show_tech_support is arista.avd.defined %}
+    policy show tech-support
```

Comment:
Command indentation should be 3 spaces. (Where as the Jinja code indentation is 4 spaces.) So, please make sure all commands are indented by only 3 spaces from every next level.